cel/src/magic.rs

```diff
@@ -12,7 +13,8 @@ impl_conversions!(
     Arc<String> => Value::String,
     Arc<Vec<u8>> => Value::Bytes,
     bool => Value::Bool,
-    Arc<Vec<Value>> => Value::List
+    Arc<Vec<Value>> => Value::List,
+    Arc<dyn Opaque> => Value::Opaque
```

Comment:
Ooops my bad... I had that diff _in the PR_, just not the commit in the PR 🤦 
Good catch, thanks! 🙏